src/Nuget/Sarif.Multitool.nuspec

```diff
@@ -38,8 +38,6 @@
           target="tools\netcoreapp3.1\any" />
     <file src="bld\bin\Signing\netstandard2.0\**"
           target="tools\netcoreapp3.1\any" />
-    <file src="bld\bin\Signing\netstandard2.1\**"
```

Comment:
>netstandard2 [](http://example.com/codeflow?start=31&length=12)

btw - the need for this change indicates some technical debt in our early build/test scripts. we could/should consider a richer build (that compiles this package and also the NPM package) that runs at PR time.